src/main/java/dev/emi/trinkets/api/event/TrinketEquipCallback.java

```diff
@@ -0,0 +1,25 @@
+package dev.emi.trinkets.api.event;
+
+import dev.emi.trinkets.api.SlotReference;
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ItemStack;
+
+public interface TrinketEquipCallback {
+    Event<TrinketEquipCallback> EVENT = EventFactory.createArrayBacked(TrinketEquipCallback.class,
+    listeners -> (stack, slot, entity) -> {
```

Comment:
This entire subsection needs to be indented further

Reply:
Thought I matched the indenting of the Drop callback exactly. Will check

Reply:
As I thought, the original Drop callback had the exact indentation I used. without an exact guide for what indentation you want, I went with this:
![image](https://github.com/emilyploszaj/trinkets/assets/72876796/dc27e4a1-966f-4d8d-9193-43de78bafbfb)

But unsure where exactly you'd want the enclosing parentheses. Could also do this:
![image](https://github.com/emilyploszaj/trinkets/assets/72876796/b5948939-c787-4c06-9f31-bc6e9a13506a)

Reply:
Yeah this is just style inconsistency in the codebase. Original is fine if it's matching some other similar class, apologies for the hard to resolve ask.

Reply:
kept original